pgmpy/factors/FactorSet.py

```diff
@@ -112,6 +129,14 @@ def product(self, factorset, inplace=True):
         factorsets: FactorSet1, FactorSet2, ..., FactorSetn
             FactorSets to be multiplied
 
+        inplace: A boolean (Default value True)
+            If inplace = True , then it will modify the FactorSet object, if False, it will
+            return a new FactorSet object.
+
+        Returns
+        --------
+        If inpalce = False, will return a new FactorSet object, which is product of two factors
```

Comment:
I think, this would be better: `FactorSet instance : If inplace = False, ....... `